UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+namespace UKHO.MaritimeSafetyInformation.Common.Extensions
+{
+    [ExcludeFromCodeCoverage]
+    public static class DateTimeExtensions
```

Comment:
Please add unit test for extension methods and remove [ExcludeFromCodeCoverage]

Reply:
remove [ExcludeFromCodeCoverage]